### PR TITLE
Add ability to use symbols for `alias_tool` and `delegate_to`

### PR DESCRIPTION
**Pull request by AlexWayfer**
And arrays of symbols.

### CODE REVIEW

File: toys-core/lib/toys/loader.rb [hunk 1]
```diff
@@ -233,7 +233,7 @@ def has_subtools?(words) # rubocop:disable Naming/PredicateName
     #
     def split_path(str)
       return str if str.is_a?(::Array)
-      @extra_delimiters ? str.split(@extra_delimiters) : [str]
+      @extra_delimiters ? str.to_s.split(@extra_delimiters) : [str]
```
Comment (dazuma) [inline toys-core/lib/toys/loader.rb]: Can you call `to_s` on the str in the no-extra-delimiters case as well? (... `: [str.to_s]`)

Comment (AlexWayfer) [reply toys-core/lib/toys/loader.rb]: Done, also for Arrays.

**Comment by dazuma** [comment]
Note: This change has been released in version 0.9.1.

**Comment by AlexWayfer** [comment]
> Note: This change has been released in version 0.9.1.

Nice, thank you!

**Pull request merged**